pkg/suggestion/cma_unit_test/main.py

```diff
@@ -0,0 +1,184 @@
+import grpc
+from concurrent import futures
+
+import time
+
+from pkg.api.python import api_pb2, api_pb2_grpc
+from pkg.suggestion.cma_unit_test.db_init import connect_db
+from pkg.suggestion.cma_unit_test.internface import create_study, get_study_config, get_suggestion_param, set_suggestion_param, \
+    update_suggestion_param, create_trial, \
+    get_suggestion_param_list, get_worker_list, get_worker, get_worker_logs, create_worker
+from pkg.suggestion.cma_unit_test.worker import spawn_worker
+
+_ONE_DAY_IN_SECONDS = 60 * 60 * 24
+
+
+class ManagerService(api_pb2_grpc.ManagerServicer):
```

Comment:
Why do you implement a new ManagerService? 
You should use [this manager](https://github.com/kubeflow/katib/blob/master/cmd/manager/main.go) .

Reply:
Yes, I am using the manager implemented by you. This one is just for (unit) testing.
Speaking about testing, it seems that I need to follow the `getting-start.md` to deploy the katib on k8 clusters. Otherwise, it will stuck on the steps relevant to kubenetes.

Reply:
OK, I understand! Thanks.